ui/src/views/Denylist/ViewDenylist/index.jsx

```diff
@@ -0,0 +1,56 @@
+import { hot } from 'react-hot-loader';
+import React, { PureComponent } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import PlusIcon from 'mdi-react/PlusIcon';
+import Denylist from './Denylist';
+import Dashboard from '../../../components/Dashboard';
+import Search from '../../../components/Search';
+import Button from '../../../components/Button';
+import HelpView from '../../../components/HelpView';
+
+@hot(module)
+@withStyles(theme => ({
+  plusIconSpan: {
+    ...theme.mixins.fab,
+  },
+}))
+export default class ViewDenylist extends PureComponent {
+  state = {
+    notificationsSearch: '',
+  };
+
+  handleNotificationsSearchSubmit = notificationsSearch => {
+    this.setState({ notificationsSearch });
+  };
+
+  handleCreate = () => {
+    this.props.history.push('/notify/denylist/add');
+  };
+
+  render() {
+    const { classes, description } = this.props;
+    const { notificationsSearch } = this.state;
+
+    return (
+      <Dashboard
+        title="Notifications Denylist"
+        helpView={<HelpView description={description} />}
+        search={
+          <Search
+            onSubmit={this.handleNotificationsSearchSubmit}
+            placeholder="Notification address contains"
+          />
+        }>
+        <Denylist searchTerm={notificationsSearch} />
```

Comment:
We don't need to create this component. We could have it here.

Reply:
@helfi92 We need to run the graphql query on every `searchTerm` state change that's why I made a separate component. I feel it's done similarly in `views/Roles/ViewRoles/`. On the other hand, components like `view/Scopes/ListScopes` don't do this because, instead of querying the server with the filter, they pass the `searchTerm` to the rendering component which then filters it. I feel passing the `searchTerm` as props is the easiest way to re execute the graphql query. What are your thoughts on this?

Reply:
Makes sense :)